__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../app');
+const connection = require('../db/connection');
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe('app', () => {
+  describe('/api/topics', () => {
+    it('200: GET returns array of topic objects, each with slug, and description properties', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+
+          expect(Array.isArray(topics)).toBe(true);
```

Comment:
If we are going to do a forEach, we will need to make sure the length of the array is expected, to ensure all data is coming through.